boards/adafruit/metro_rp2350/board.cmake

```diff
@@ -16,3 +18,4 @@ board_runner_args(uf2 "--board-id=RP2350")
 
 include(${ZEPHYR_BASE}/boards/common/uf2.board.cmake)
 include(${ZEPHYR_BASE}/boards/common/openocd.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/probe-rs.board.cmake)
```

Comment:
Sorting this would involve out of scope changes. Avoided for this reason as generally maintainers are opposed of format changes in functional change PRs